readme.md

```diff
@@ -133,6 +133,17 @@ embeddings for many languages. Here's a list of resources;
 
 - [AraVec](https://github.com/bakrianoo/aravec#download) has embeddings for Arabic trained on twitter and/or Wikipedia.
 
+## **Sparse Featurizers**
+
+### SemanticMapFeaturizer
+
+**`rasa_nlu_examples.featurizers.sparse.SemanticMapFeaturizer` [docs](docs/featurizer/semantic_map/)**
+
+The `SemanticMapFeaturizer` is an experimental *sparse* featurizer developed by Rasa.
+It can only be used in combination with pre-trained embedding files, which you can
+find [here](https://github.com/RasaHQ/rasa-embeddings/tree/main/embeddings/semantic_map).
+Please refer to our [blog posts](blog.rasa.com/exploring-semantic-map-embeddings-1/) for more details.
```

Comment:
1. Is this blog live yet? 
2. blog.rasa.com/exploring-semantic-map-embeddings-1/ -> https://blog.rasa.com/exploring-semantic-map-embeddings-1/

Reply:
The blog is [drafted](https://blog.rasa.com/p/f7d30739-f59a-409f-abe6-39b1c6c657f7/) on Ghost. Karen said we'll publish on Monday, but I don't know which Monday ;D

Reply:
> Why aren't we using `FileNotFoundError` found in base python?

I have no idea. Just following what we do in Rasa. Should I change it?

Reply:
I'm changing it.